d2/src/main/java/com/linkedin/d2/balancer/D2ClientBuilder.java

```diff
@@ -201,7 +201,8 @@ public D2Client build()
                   _config.serviceDiscoveryEventEmitter,
                   _config.dualReadStateManager,
                   _config.xdsExecutorService,
-                  _config.xdsStreamReadyTimeout
+                  _config.xdsStreamReadyTimeout,
+                  _config.loadBalancerExecutor
```

Comment:
to be more descriptive, "dualReadNewLbExecutor" would be better?

Reply:
Thanks. Updated~